src/Parlot/Fluent/Between.cs

```diff
@@ -17,11 +18,19 @@ public Between(Parser<A> before, Parser<T> parser, Parser<B> after)
             _after = after ?? throw new ArgumentNullException(nameof(after));
         }
 
+        public bool CanSeek => _before is ISeekable seekable && seekable.CanSeek;
```

Comment:
Extension methods with aggressive inline to reduce duplication?

Reply:
I am not concerned about perf here since this is called once per application (when the parser tree is built). How would an extension method remove duplication?